_scripts/pages/store/index.js

```diff
@@ -47,8 +47,32 @@ Promise.all([jQuery, modal]).then(([$]) => {
             $trigger.click()
 
             ga('send', 'event', 'Store', 'View Product', $item.data('product-name'))
+
+            updateShippingEstimate($item.attr('data-printful-id'))
         })
 
+        /**
+         * updateShippingEstimate
+         * Fetches a psudeo shipping estimate
+         *
+         * @param {String} printfulId - the variant printful_id
+         *
+         * @return {Void}
+         */
+        var updateShippingEstimate = function (printfulId) {
+            var $m = $('.js-leanmodal-active .modal__shipping')
+            // GET /api/gelocate
+            // with the parameters "shipping" and "item" (printful variant id)
+            $.getJSON(baseUrl + 'api/geolocate?shipping&item=' + printfulId, function (data) {
```

Comment:
```js
            $.getJSON(baseUrl + 'api/geolocate?shipping&item=' + printfulId, function (data) {
                // Update price information
				try {
				  const cost = data['shipping']['estimates'][0]['cost']

				  if (cost != null) {
                    $m.text('+ $' + cost + ' estimated shipping')
                  }
                } catch (e) {}
            })
```

I would use a catch block here instead. That way it does not fail if a nested value does not exist